Dockerfile.local

```diff
@@ -0,0 +1,9 @@
+FROM gcr.io/distroless/static:debug
+
+WORKDIR /
+COPY bin/k8s .
```

Comment:
nit: we've decided to rename the provisioner to `registry-v1` in #100 so depending which lands first this may have to be rebased to point to the new binary.

Reply:
I see.  I'll change to plain-v0.  Thanks!